make_go.sh

```diff
@@ -1,5 +1,5 @@
 set -eu
 
-cargo build
+cargo build --release
```

Comment:
assuming this was in coordination w/ adding the `cgo` flags stuff. should we keep this?